changelog.json

```
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.9.4-beta1": [
+      "[Added] Add a menu item to view a committed file change on GitHub - #12492. Thanks @haykam821!",
+      "[Added] Customizable high contrast theme - #12833",
+      "[Fixed] App no longer crashes on successful merge after conflicts resolved - #12888",
+      "[Improved] Use same width for hunk expansion handles as we do for line numbers - #12887",
+      "[Improved] Small fixes/improvements in diff expansion - #12882"
```

Maybe we can be a bit more specific about this one? Also I think it should be a fix.
```suggestion
      "[Fixed] Use the correct icon for diff expansion - #12882"
```